docs/usage.md

```diff
@@ -269,6 +269,30 @@ With this configuration in place deployment should happen automatically every ti
 
 There is also a [Shiny server application](https://posit.co/download/shiny-server/), which you can install on your own infrastruture and use to host applications yourself.
 
+## Gene set enrichment analysis
+
+Currently, two tools can be used to do gene set enrichment analysis.
+
+### GSEA
+
+[GSEA](https://www.gsea-msigdb.org/gsea/index.jsp) tests for differential genes from within a user-provided set of genes; this requires a GMT or GMX file. The following example shows how to enable this:
+
+```bash
+--gsea_run true \
+--gene_sets_files gene_sets.gmt
+```
+
+### g:Profiler
+
+The [gprofiler2](https://cran.r-project.org/web/packages/gprofiler2/vignettes/gprofiler2.html) package can be used to test which pathways are enriched for genes that have been determined to be differentially expressed; it is an R interface for the g:Profiler webtool. In the simplest form, this feature can be enabled with the parameters from the following example:
```

Comment:
> test which pathways are enriched for genes that have been determined to be differentially expressed

Is this true? Or is it rather looking for which pathways are enriched in the set of differential genes relative to the background? Not quite the same thing.

Reply:
Umm, I'm not sure I understand. I was meaning that gprofiler searches for enrichment of pathways with DE genes as it only receives those (it is given the filtered matrix, not the full results from DESeq2/limma).

Reply:
So, your wording suggests that you start with a gene set and try to understand whether there are more differentially expressed genes that you would expect by chance. I suspect that's not what's happening.

Rather, we take the differentially expressed gene list, and look to see if there are more members of each gene set than we would expect by chance, relative to the background. The background contains all the genes passed to d/e, so it effectively does have the full results from d/e (though all it knows is that they are not significant).

Reply:
Aah, okay, maybe I formulated that badly. I wanted to indicate that only those genes are piped into the enrichment analysis that DESeq/limma deemed differential. Maybe something like this instead?

`The [gprofiler2](https://cran.r-project.org/web/packages/gprofiler2/vignettes/gprofiler2.html) package can be used to test which pathways are enriched for genes that the DESeq2/limma modules determined to be differentially expressed; it is an R interface for the g:Profiler webtool.`

Reply:
That has the same ambiguity to me. How about something like:

The [gprofiler2](https://cran.r-project.org/web/packages/gprofiler2/vignettes/gprofiler2.html) package can be used to test which pathways are enriched in the set of differential genes produced by the the DESeq2 or limma modules.  It is an R interface for the g:Profiler webtool.

Reply:
That sounds good, thanks!